sklearn/tests/test_grid_search.py

```diff
@@ -768,3 +768,53 @@ def test_parameters_sampler_replacement():
     sampler = ParameterSampler(params_distribution, n_iter=7)
     samples = list(sampler)
     assert_equal(len(samples), 7)
+
+
+_atleast_py34 = (sys.version_info.major == 3 and
+                 sys.version_info.minor >= 4)
+
+
+@np.testing.decorators.skipif(not _atleast_py34)
+def test_grid_search_backend():
+    # copied from test_grid_search
+    from multiprocessing import pool
+    backend = pool.get_context(method="forkserver")
```

Comment:
Yes, AFAIK, for using it out of the box like this. I suppose you could roll your own somehow context somehow which is why I didn't mention it in the docstring. Maybe why it's not mentioned for joblib either. I just happened to find it supported reading the source.  https://docs.python.org/3/whatsnew/3.4.html#whatsnew-multiprocessing-no-fork